nodes/object_operators/ScFindNearest.py

```diff
@@ -27,12 +33,48 @@ def error_condition(self):
             super().error_condition()
             or self.inputs["Distance"].default_value < 0.0
         )
-    
+
+    def functionality(self):
```

Comment:
You can skip `functionality()` completely if you only need to calculate and set output values. Since it doesn't return `out` variable, moving this to `post_execute()` would be convenient. Then there will be no need for `info` global var.

Reply:
I moved it because I wasn't sure if the "functionality" method was optimized by the graph in any way (parallelism / out of order execution). I was thinking of the performance of the full vertex search on very large meshes and decided to err on the side of caution.

It is easy to move and can go ahead with that.